src/app/problem/right/OutputMatchPolicy.js

```diff
@@ -0,0 +1,53 @@
+import Block from "./Block";
+import {useState} from "react";
+import {EditSaveCancelButton} from "../commons/EditSaveCancelButton";
+import {useTags} from "../../usecases/TagUseCase";
+import {TextInputField} from "../../commons/TextInputForm/TextInputField";
+import {TextInputItems} from "../../problem-old/edit/TextInputItems";
+import TagWithIconList from "../commons/TagWithIconList";
```

Comment:
Move `TextInputItems` to `/problem/commons` package will be better! 
Do not leave it in the `problem-old` package.